client/environment_test.go

```diff
@@ -45,11 +45,88 @@ var _ = Describe("Environment Client", func() {
 			})
 		})
 
+		Describe("SuccessMultiPages", func() {
```

Comment:
without project or organization the http call should fail

Reply:
It doesn't fail (this applies to the code before my changes).
I guess it uses organization by default if nothing is passed.